src/zenml/cli/model.py

```diff
@@ -267,7 +267,7 @@ def update_model(
             trade_offs=tradeoffs,
             ethics=ethical,
             limitations=limitations,
-            tags=tag,
+            add_tags=tag,
```

Comment:
I would expect this to take a list of strings not just one string

Reply:
It is a list, just the limitation of click - the name of arg must match the name of arg in the function and the user pass it as `-t a -t b ...`